yt/visualization/volume_rendering/scene.py

```diff
@@ -364,6 +364,7 @@ def save_annotated(
         self,
         fname: Optional[str] = None,
         label_fmt: Optional[str] = None,
+        label_fontsize: int = 10,
```

Comment:
We can't add allowed-positional arguments to public functions in arbitrary places in the signature because it might break existing user code. Since everything here is currently allowed-positional, any new argument must be appended at the end. And because allowed-positional are a nightmare to deprecate if they ever need to be, let's make it keyword-only.

Reply:
I don't really use type hints, so I messed this up.  Does this mean I should add `Optional[]`?

Reply:
No, your type hint is perfect (unless floats are also valid, in which case the annotation should reflect it). For reference, `Optional[T]` means "can be `T` or `None`". It's not needed here because the default value already has the proper type.

What I'm advocating for here is making new arguments keyword-only.

Reply:
updated